test/performance/lib/performance/result.rb

```diff
@@ -6,8 +6,8 @@
 
 module Performance
   class Result
-    attr_reader :test_name, :measurements, :tags, :timer, :artifacts
-    attr_accessor :exception, :iterations
+    attr_reader :test_name, :measurements, :tags, :timer, :artifacts, :exception
```

Comment:
I'm surprised there isn't a "these should be alphabetized" cop in play.

Reply:
It might be on the list of disabled cops that we didn't get to yesterday!